File: app/src/debug/java/com/jenshen/smartmirror/app/MyDebugSmartMirrorApp.java
package com.jenshen.smartmirror.app;

import com.facebook.stetho.Stetho;
import com.uphyca.stetho_realm.RealmInspectorModulesProvider;

public class MyDebugSmartMirrorApp extends SmartMirrorApp {

    @Override
    public void onCreate() {
        super.onCreate();
        Stetho.initialize(
                Stetho.newInitializerBuilder(this)
                        .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
                        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
                        .enableWebKitInspector(RealmInspectorModulesProvider.builder(this).build())
                        .build());
    }
}

File: app/src/debug/java/com/jenshen/smartmirror/app/MyDebugSmartMirrorApp.kt
package com.jenshen.smartmirror.app

import com.facebook.stetho.Stetho

class MyDebugSmartMirrorApp : SmartMirrorApp() {

    override fun onCreate() {
        super.onCreate()
        Stetho.initialize(
                Stetho.newInitializerBuilder(this)
                        .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
                        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
                        //.enableWebKitInspector(RealmInspectorModulesProvider.builder(this).build())
                        .build())
    }
}